README.md

```diff
@@ -1,33 +1,64 @@
-# HVNC proposals and discussions
+# HGVS Nomenclature
 
-**Under construction**
+This repo is the home of the HGVS Nomenclature maintained by the Human Genome Organization Variation Nomenclature Committee.
 
-More instructions will follow.
+## Have Questions?
 
+Please use the [Discussions](discussions) forum to search for answers and ask new questions.
 
-## Proposals
+## Proposal Process Overview
 
-### New proposal
+- Anyone may submit a proposal for changes to [Discussions](discussions). The proposal is initially considered OPEN and under discussion.
+- Anyone may comment on an OPEN proposal.
+- The HVNC will evaluate all proposals from time to time. For each proposal, the committee may choose to:
+  - leave the proposal OPEN for further discussion;
+  - mark the propossal as REJECTED with explanation if the proposal is deemed unworkable, incomplete, or inconsistent with the existing guidelines, or if it is similar to existing or recently considered proposals; or
+  - move the propsal to DRAFTING to indicate that a member of the HVNC will understake drafting of the changes.
+- When a proposal is in the DRAFTING stage, a GitHub issue is created to track writing for the intended documentation change. Writing will be undertaken in a branch dedicated for the assigned issue. A pull request will eventually be submitted and evaluated.
+- Evaluation of the PR is limited to HVNC members (i.e., not for public discussion), although specific users may be asked for comments when warranted. If the PR is eventually merged, the discussion proposal is marked as ADOPTED.
+- If the PR is not merged for any reason (e.g., if we realize that the PR creates unforeseen issues with the existing nomenclature), the proposal will either be re-opened for more discussion or rejected at the discretion of the committee.
```

Comment:
Are we going to have close discussions once complete. Perhaps if we are, come comments about finding and navigating them to help those not used to git

Reply:
Discussions are open to the public, even with out a github account I believe. So, I think the barrier is about as low as it can be.